gcalbot/gcalbot/http.go

```diff
@@ -33,10 +36,32 @@ func NewHTTPSrv(
 	h.OAuthHTTPSrv = base.NewOAuthHTTPSrv(kbc, config, requests, h.db, h.handler.HandleAuth,
 		"gcalbot", base.Images["logo"], "/gcalbot")
 	http.HandleFunc("/gcalbot", h.healthCheckHandler)
+	http.HandleFunc("/gcalbot/home", h.homeHandler)
+	http.HandleFunc("/gcalbot/image/screenshot", h.screenshotHandler)
 	http.HandleFunc("/gcalbot/events/webhook", h.handleEventUpdateWebhook)
 	return h
 }
 
 func (h *HTTPSrv) healthCheckHandler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "OK")
+	_, _ = fmt.Fprintf(w, "OK")
+}
+
+func (h *HTTPSrv) homeHandler(w http.ResponseWriter, r *http.Request) {
+	homePage := `Google Calendar Bot is a <a href="https://keybase.io">Keybase</a> chatbot
+	which connects with your Google calendar to notify you of invites, upcoming events and more!
+	<div style="padding-top:10px;">
+		<img style="width:300px;" src="/gcalbot/image/screenshot">
+	</div>
+	`
+	if _, err := w.Write(base.MakeOAuthHTML("gcalbot", "home", homePage, "/gcalbot/image/logo")); err != nil {
```

Comment:
how is `/image/logo` handled?

Reply:
handled by the oauth package: https://github.com/keybase/managed-bots/blob/bf7bbab87a3932f7244ef71771b3c775b3f3bae4/base/oauth.go#L56